README.md

```diff
@@ -1,53 +1,7 @@
 # Simulated Devices
 
-`devices.json` is a catalog of common web-enabled devices and their properties.
-
-    {
-      TYPES: [ "phones", … ],
-      phones: [
-        {
-          name: "LePhone",
-          width: 360,
-          height: 640,
-          pixelRatio: 3,
-          userAgent: "Mozilla/5.0 (Mobile; rv:39.0) Gecko/39.0 Firefox/39.0",
-          featured: true,
-          touch: true
-        }
-      ], …
-    }
-
-## Device properties
-
-- `name` brand and model(s).
-- `width` viewport width.
-- `height` viewport height.
-- `pixelRatio` ratio from viewport to physical screen pixels.
-- `userAgent` UA string of the device's browser.
-- `touch` whether it has a touch screen.
-- `os` the OS type it usually ships with.
-- `featured` if set to true, the device appears in the RDM drop down list (Firefox UI) by default
-
-## Device types
-
-- `phones` (smart phones)
-- `tablets`
-- `laptops`
-- `televisions`
-- `consoles` (gaming consoles)
-- `watches` (smart watches)
-
-## Downloads
-
-- Stable: [Mozilla CDN](https://code.cdn.mozilla.net/devices/devices.json).
-- Latest: [GitHub](https://raw.githubusercontent.com/mozilla/simulated-devices/master/devices.json).
-
-## Contribute
-
-- Use the `check.html` helper.
-- Use `node test.js` to verify your `devices.json`.
-- Submit [pull requests](https://github.com/jankeromnes/devices/pulls).
-
+`devices.json` is a catalog of common web-enabled devices and their properties, that was previously used by the Responsive Design Mode in Firefox DevTools.
+The data now lives in Mozilla's RemoteSettings `devtools-devices` collection, see https://firefox-source-docs.mozilla.org/devtools/responsive/devices.html for more information
 ## Sources
 
 - Chrome Devtools: [emulated_devices/module.json](https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/front_end/emulated_devices/module.json).
```

Comment:
Why not delete this as well?

Reply:
I wanted to keep it for future reference, just in case, but it'd probably be better to just remove it and add the link into firefox doc